.github/workflows/integration-test-config/action.yaml

```
@@ -28,6 +28,7 @@ runs:
     shell: bash --noprofile --norc -eo pipefail -x {0}
     run: tests/scripts/github-action-helper.sh print_k8s_cluster_status
 
+  # This setp is not required for canary integration tests so keeping this new file
```

Just to be clear, is this valid?
```suggestion
  # This step is required for all the integration tests except the canary tests
```

---

yes